### PR TITLE
Kba/conditional render

### PR DESCRIPTION
**Pull request by KasperBaun**
Main changes:
- Added '<ConditionalRender /> as a new feature
- Cleaned up some unused files and refactored inputprops to be more aligned in namingconvention and to be more reusable
- Optimized render-flow by refactoring '<RowRenderer />, '<ColumnRenderer /> and so into "components/renderers"
- Did a few module.css converts into React.CSSProperties to prepare for CSS-IN-JS conversion later in development

### CODE REVIEW

File: packages/core/src/components/renderers/column-renderer/ColumnRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+export const ColumnRenderer: React.FC<ColumnRendererProps> = ({ column, questions }) => {
+
+    const logger = resolveQuickFormService("logger");
+    logger.log("Rendering column {@column}", column);
+
+    if (column.type === "question") {
+        const question = findQuestionByLogicalName(column.ref!, questions);
+        if (!question) return null;
+        if (question.visible && question.visible?.rule) {
+            return <ConditionalRender key={question.logicalName} style={fullRowStyle} model={question} />
```
Comment (pksorensen) [inline packages/core/src/components/renderers/column-renderer/ColumnRenderer.tsx:31]: We could make ConditonalRender general and instead use children, so we do 
```
<ConditionalRender enine={question.visible.engine} rule={question.visible.rule}>
  <Question key={question.logicalName} style={fullRowStyle} model={question} />
</ConditionalRender>
```
That why conditionalRender do not need to know what it need to handle what its rendering, only validating the rule with a given engine and then just <>{children}</>. 

Does that make sense, is it better? (Have not looked inside conditionalrender yet.

What happens if we move the logic to the model? is there even a need for conditional render then? as we can simply do if (question.visible === false) return null;

File: packages/core/src/components/renderers/conditional-render/ConditionalRender.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    // function evalInScope(js: string, contextAsScope: any) {
+    //     return new Function(`with (this) { return (${js}); }`).call(contextAsScope);
+    // }
+
+    interface Context {
+        [key: string]: any;
+    }
+
+    function functionInScope(js: string, context: Context): boolean {
```
Comment (pksorensen) [inline packages/core/src/components/renderers/conditional-render/ConditionalRender.tsx:28]: The function should be defined outside the component, should it not? performance from defining the function on each render (i know its minimal), it also cleans up the code i think

**Review by pksorensen — COMMENTED** [review]
I think we just merge it in and we can discuss my comments and change after. Looks good, rather large one to review with a lot of refactoring, making it abit hard to see if i missed some logic/changes that are important. Early on i think we merge and fix issues that emerge, later on we should keep refactoring and logic/features in seperate PRs to make it easier to review.

LGTM

File: packages/core/src/components/renderers/conditional-render/ConditionalRender.tsx (hunk 2 above)
Comment (KasperBaun) [reply packages/core/src/components/renderers/conditional-render/ConditionalRender.tsx]: Implemented suggested changes in new commit.

File: packages/core/src/components/renderers/column-renderer/ColumnRenderer.tsx (hunk 1 above)
Comment (KasperBaun) [reply packages/core/src/components/renderers/column-renderer/ColumnRenderer.tsx]: Implemented suggested changes in new commit.

**Review by pksorensen — COMMENTED** [review]
Nice

**Pull request merged**